docs/required-fields.md

```diff
@@ -24,6 +24,25 @@ spec:
   maturity: The operator's maturity level. This field can be omitted until a process to get this value is defined at a later date.
 ```
 
+## Categories
+
+For the best user experience, choose from the following categories:
+
+| Category  |
+|-----------|
+| AI/Machine Learning |
+| Big Data |
+| Security |
+| Networking |
+| Storage |
+| Integration & Delivery |
+| Database |
+| Cloud Provider |
+| Logging & Tracing |
+| Streaming & Messaging |
+| Monitoring |
+| OpenShift Optional |
+
 ## Example CSV
```

Comment:
Can you update the example CSV below? The category has `openshift optional` rather  than `OpenShift Optional`.